library/options.json

```diff
@@ -4,6 +4,10 @@
       "type": "string",
       "default_value": "+Y"
     },
+    "up_direction2": {
```

Comment:
please add a test in TestSDKOption.cxx

Reply:
TestSDKOptions.cxx test all types of options, you are adding a new type of option, you should test it.